src/components/date-picker/flatpickr-adapter/flatpickr-adapter.scss

```diff
@@ -7,6 +7,7 @@
 
 /**
  * @prop --today-label: Tooltip label for "today".
+ * @prop --limel-date-picker-dropdown-font-family: Font family override for the date-picker dropdown. Because the dropdown is rendered in a portal, it inherits its font from the body element. If the body's font is not what should be used for the date-picker, this property can be used to override the font.
```

Comment:
this one is private prop, but it's publicly documented. See [`968fe8b` (#2673)](https://github.com/Lundalogik/lime-elements/pull/2673/commits/968fe8b15b84b6c3cdda6e9ce0c24f81b9e26020#r1442645698)

Reply:
Removed.